packages/locale/src/js-joda-locale.js

```diff
@@ -5,9 +5,12 @@
 import { use } from '@js-joda/core';
 import plug from './plug';
 import Locale from './Locale';
+import { WeekFields, ComputedDayOfField } from './temporal/WeekFields';
 
 use(plug);
 
 export {
-    Locale
+    Locale,
+    WeekFields,
+    ComputedDayOfField
```

Comment:
Its still exported ?

Reply:
Thanks @pithu - tests in and export removed.  Hopefully the tests are of the sort your require.